test/rest_request_template/travis_correct_auto_table_creation.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Thanks for adding Unit and End to end test. 
Can you also add IT test specifically testing the newly added code in Record Service? 

You can take a look at testNativeAvroInputIngestion test in SinkServiceV2IT test which will test your newly added code. 
Two reasons to add them:
1. Integration test is fastest way to check your expected behavior without hoping your kafka and kafka connector works or not. Although your End to end test will cover the behavior but it wont give you fast feedback. 
2. Code coverage.

Reply:
You will only have to add a new schematization config in your new test.